### PR TITLE
[WIP] Update template to use main loop approach

### PR DESCRIPTION
**Pull request by vontell**
Still need to do a quick test. This updates the existing template to use the new main loop approach from @RG-nAmKcAz

### CODE REVIEW

**Comment by nAmKcAz** [comment]
probably want to checkin the `tsconfig.json`

**Review by nAmKcAz — APPROVED** [review]
Approved

**Comment by vontell** [comment]
Tested on prod and this works, so should be good to go once we get the next approval.

File: tsconfig.json
Comment (abeizer) [inline tsconfig.json]: Does this need to be `index.ts`?

**Review by abeizer — APPROVED** [review]

**Pull request merged**
